keuzewijzer-hbo-deeltijd-ict-API/Models/Role.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace keuzewijzer_hbo_deeltijd_ict_API.Models
+{
+    public class Role
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public List<User> Users { get; set; }
```

Comment:
Hoort hier een lijst van users?

Reply:
Fixed

Reply:
Volgens mij niet, de build faalt, ik denk door je seeder data.
Als het teveel werk is mag het van mij wel weer terug

Reply:
Toch handig de Github Actions 😄

Reply:
Had m ook gelijk door, al gefixed met een 2e commit